source/Playnite/Localization/LocSource.xaml

```diff
@@ -1172,6 +1172,8 @@ SHIFT-ENTER: open item menu</sys:String>
     <sys:String x:Key="LOCEmuOverridePlatformTooltip">When set, scanner will assign this platform to all games, overwriting any automatically detected platforms.</sys:String>
     <sys:String x:Key="LOCSearchIncludeCommandsInDefault">Include commands in default search</sys:String>
     <sys:String x:Key="LOCSearchIncludeCommandsInDefaultTooltip">When disabled, commands won't be included in default search until # prefix is used.</sys:String>
+    <sys:String x:Key="SearchBoxSearchWithAcronyms">Search using acronyms</sys:String>

```

Comment:
In my opinion this should be called something like `Support search using acronyms`. At first glance it looks like the search will be done only using acronyms when enabled.

Reply:
Changed